src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  // filter Men, and set condition for century description
+  const filterMen = people.filter(
+    men => men.sex === 'm'
+    && (!century || Math.floor((men.died / 100) + 1) === century));
```

Comment:
```suggestion
  const filterMen = people.filter(person =>
    century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : (person.sex === 'm')
  );
```
Можна записати трошки лаконічніше. (це стосується також і наступних двох функцій)